source/GStreamerWebAudioPlayerClient.cpp

```diff
@@ -253,7 +253,7 @@ void GStreamerWebAudioPlayerClient::pushSamples()
     {
         return;
     }
-    uint32_t availableFrames = 0u;
+    uintptr_t availableFrames = 0u;
```

Comment:
Can you explain why you change uint32_t to uintptr_t?

Reply:
The change is done to uintptr_t, as 64 bit build doesn't support uint32_t .